features/admin_creates_auction.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Admin creates auctions in the admins panel
+  As an admin
+  I should be able to create auctions
+
+  Scenario: Adding an auction
+    Given I am an administrator
+    And there is a tock project to bill to
```

Comment:
I wonder if this should be swept into a more generic setup block, since it doesn't seem to relate to the user interface.